LEGO1/omni/include/mxstring.h

```diff
@@ -13,19 +14,31 @@ class MxString : public MxCore {
 
```

Comment:
Can you move the declaration of this `operator=` down to the other ones? It should also return `MxString&` instead of a const-reference. Then re-order the declarations of the constructors to be consistent with other class declarations as well. Thank you!